CMakeLists.txt

```diff
@@ -1,8 +1,11 @@
-cmake_minimum_required(VERSION 3.7.0)
+cmake_minimum_required(VERSION 2.6)
```

Comment:
Shouldn't this be `2.8.3` given the comment in `FindICU.cmake` stating `CMake >= 2.8.3 (for new version of find_package_handle_standard_args)`?